examples/unet_segmentation_3d.ipynb

```diff
@@ -146,90 +123,336 @@
     "    num_res_units=2,\n",
     ")\n",
     "\n",
-    "loss = networks.losses.DiceLoss(do_sigmoid=True)\n",
+    "loss = monai.losses.DiceLoss(do_sigmoid=True)\n",
     "opt = torch.optim.Adam(net.parameters(), lr)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 14,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
    "source": [
-    "trainEpochs = 30\n",
+    "# Since network outputs logits and segmentation, we need a custom function.\n",
+    "def _loss_fn(i, j):\n",
+    "    return loss(i[0], j)\n",
     "\n",
-    "loss_fn = lambda i, j: loss(i[0], j)\n",
+    "# Create trainer\n",
     "device = torch.device(\"cuda:0\")\n",
-    "\n",
-    "trainer = create_supervised_trainer(net, opt, loss_fn, device, False)\n",
-    "\n",
-    "checkpoint_handler = ModelCheckpoint('./', 'net', n_saved=10, require_empty=False)\n",
-    "trainer.add_event_handler(\n",
-    "    event_name=Events.EPOCH_COMPLETED,\n",
-    "    handler=checkpoint_handler,\n",
-    "    to_save={'net': net}\n",
-    ")\n",
+    "trainer = create_supervised_trainer(net, opt, _loss_fn, device, False,\n",
+    "                                    output_transform=lambda x, y, y_pred, loss: [y_pred, loss.item(), y])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### optional section for checkpoint and tensorboard logging\n",
+    "# adding checkpoint handler to save models (network params and optimizer stats) during training\n",
+    "checkpoint_handler = ModelCheckpoint('./runs/', 'net', n_saved=10, require_empty=False)\n",
+    "trainer.add_event_handler(event_name=Events.EPOCH_COMPLETED,\n",
+    "                          handler=checkpoint_handler,\n",
+    "                          to_save={'net': net, 'opt': opt})\n",
+    "train_stats_handler = StatsHandler()\n",
+    "train_stats_handler.attach(trainer)\n",
     "\n",
     "@trainer.on(Events.EPOCH_COMPLETED)\n",
     "def log_training_loss(engine):\n",
-    "    print(\"Epoch\", engine.state.epoch, \"Loss:\", engine.state.output)\n",
+    "    # log loss to tensorboard with second item of engine.state.output, loss.item() from output_transform\n",
```

Comment:
perhaps put some instructions here about the training curves/prediction visualisations using `tensorboard --logdir  runs/` after running the trainer

Reply:
Added tensorboard display (inside notebook) as new section.